packages/core/src/components/popover/popover.md

```diff
@@ -1,281 +1,73 @@
-@# Popovers
```

Comment:
oh there's some very useful stuff in here that we don't want to delete.

Reply:
Yep, adding it back in Part 4.

Reply:
wait what's part 3? is it possible to just not delete it now?